homeassistant/components/discovery/__init__.py

```diff
@@ -140,6 +142,13 @@
         ignored_platforms = []
         enabled_platforms = []
 
+    for platform in enabled_platforms:
+        if platform in DEFAULT_ENABLED:
+            logger.warning(
+                'Deprecated config for discovery: service %s is enabled by default - ignoring',
```

Comment:
We're not ignoring it, right? The platform will have discovery activated. The user can remove the option though.

Reply:
I guess the wording could be clearer. What i'm trying to say to the user is:

 * You are explicitly requesting the service X be discovered.
 * We would have done discovery for that anyway.
 * So we are ignoring your request as it is a no-op - you shouldn't put that in your config any more.
 * But we will still discover this service because we don't want to break your config for something we can safely just ignore.

What about:

 ```
"You don't need to add %s to your discovery.enable configuration any more as it is enabled by default."
```

Or a call to action:

 ```
"Please remove %s from your discovery.enable configuration as it is now enabled by default."
```

Or we could just remove the warning entirely?

Reply:
I like the last wording. I think it's good to inform the user of misconfiguration even when we handle it for them.

Reply:
Ok - have pushed that wording, noticed i was using the wrong quote style and fixed that too.